lang/LangSource/PyrObject.cpp

```diff
@@ -1240,7 +1240,6 @@ void buildBigMethodMatrix()
 #else
 	size_t numentries = fillClassRows(class_object, bigTable, pool);
 #endif
-	//post("\tnumentries = %lu / %d = %.2g\n", numentries, bigTableSize, (double)numentries/(double)bigTableSize);
```

Comment:
I have back in the day actually maxed out with too many classes. The numentries tells you how many classes there are and probably could tell you if you overflow the method lookup table. I know it's not perfect right now and it's strange to read, but let's not throw it away. A verbose mode would be fine.